Component/List/list.css

```diff
@@ -0,0 +1,37 @@
+@import url("../../Utility CSS/navbar.css");
+
+.code-container {
+  width: 100%;
+}
+.box {
+  height: 30rem;
+  width: 100%;
+  padding: 1rem;
+}
+
+.list-item {
+  width: 30rem;
+  font-size: 4rem;
+  margin-left: 1rem;
+  padding: 1rem;
+  border-bottom: 1px solid var(--primary-color);
+}
+
+.list-item:hover {
+  background-color: rgba(205, 104, 245, 0.2);
+}

```

Comment:
Use color variables

Reply:
i used color variable every where but this was a special case and used only here so didnt used color variables